frontend/i18n-scripts/i18n-to-po.js

```diff
@@ -1,15 +1,26 @@
 const fs = require('fs');
 const path = require('path');
 const minimist = require('minimist');
-const { i18nextToPot } = require('i18next-conv');
+const { i18nextToPo } = require('i18next-conv');
 const common = require('./common.js');
 
-function save(target) {
-  return (result) => {
-    fs.writeFileSync(target, result);
-  };
+function addLanguageToFile(filePath, language) {
+  const data = fs.readFileSync(filePath, 'utf-8');
+  const arr = data.split('\n');
+  const index = arr.findIndex((i) => i.includes('Plural-Form'));
+  arr.splice(index, 0, `"Language: ${language}\\n"`);
```

Comment:
Set `options.language` instead of doing it this way:

https://github.com/i18next/i18next-gettext-converter/blob/master/lib/json2gettext.js#L106-L108

Reply:
I think there's a bug. I'm already setting the language option here: https://github.com/openshift/console/pull/7030/files#diff-d6aa690af9a93d13d9031bfc763234fdf51c5083c5ed52db1cf376b9c4c3d154R59, but the header isn't generated.

Reply:
@rebeccaalpert But you aren't passing an `options` parameter to that call. You need a third parameter with `{ language: language }`

Reply:
Perfect! Missed that, thanks.